Contact.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Here we import the Boostrap CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css"> 
+    <link rel="stylesheet" href="./css/contact.css"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+    <title>Responsive grid project</title>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top p-4">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="./index.html"><b>Maya Nelson</b> / PROJECT MANAGER</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse " id="navbar">
+          <ul class="navbar-nav ms-auto">
+            <li class="nav-item">
+              <a class="nav-link active"  href="./index.html">ABOUT ME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active"  href="./Resume.html">RESUME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" href="./Projects.html">PROJECTS</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="./Contact.html">CONTACT</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div id="navholder"></div>
+
+    <main>
+      <div>
+        <h1 class="d-flex justify-content-center my-5 py-3">Let's talk</h1>
+      </div>
+      <div class="d-flex justify-content-center">
+        <form class="card shadow border-0 rounded-0 mb-5 p-5 pb-1 " style="width: 680px; height: 700px;">
+          <!-- 2 column grid layout with text inputs for the first and last names -->
+        <div class="row mb-4">
+          <div class="col">
+            <div class="form-outline">
+              <label class="form-label" for="form3Example1">First name</label>
+              <input type="text" id="form3Example1" class="form-control" />              
```

Comment:
Don't rely (only) on Bootstrap to style these fields. The design has a bottom black border only, which turns blue on focus.